benchmark_test.go

```diff
@@ -28,9 +28,6 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	var exitCode int
-	defer os.Exit(exitCode)
```

Comment:
You may also use anonymous function to still have defer:
```golang
defer func(){
  os.Exit(exitCode)
}()
```
But in this case it's worth setting `exitCode = 1` by default. Sounds more complex than current approach, but wanted to mention it just in case :)